plugin/src/main/java/org/wildfly/plugin/dev/DevMojo.java

```diff
@@ -342,6 +342,14 @@ public class DevMojo extends AbstractServerStartMojo {
     @Parameter(alias = "channels")
     private List<ChannelConfiguration> channels;
 
+    /**
+     * Specifies the name used for the deployment.
+     *
+     * When the deployment is copied to the server, it is renamed with this name.
+     */
+    @Parameter(property = PropertyNames.DEPLOYMENT_NAME)
```

Comment:
I'm on my phone so it's hard to see, but if the run goal has this field, we should remove it from there. Or just move this field into the `DevMojo`.

Reply:
yes it has but not the StartMojo also the RunMojo has runtimeName which is deperecated in the PackageServerMojo

Reply:
@jamezp I don't see the point of adding this field to the 'parent' class AbstractServerStartMojo and it doesn't make sense in the context of StartMojo. So I don't think we should 'move' it out :)